integration/test/metrics_number_dimension/resources/30_dimension/config.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This is not a blocker, but it makes more sense in my head to not include the 30 and 35 dimensions configs at all in this PR, and then just introduce these configs when merging the other one.